app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/SignUpCodeFragment.kt

```diff
@@ -67,8 +67,16 @@ class SignUpCodeFragment : Fragment() {
                         nextState = actionResult.nextState
                     )
                 }
-                is SignUpResult.AttributesRequired,
+                is SignUpResult.AttributesRequired -> {
+                    // Please double-check the user flow settings to ensure that the user flow is not set to collect the required attributes.
+                    // If you'd like to collect the required attributes here, you can complete the sign up by continuing to submit the required attributes - actionResult.nextState.submitAttributes(attributes)
+                    // https://learn.microsoft.com/en-us/entra/external-id/customers/tutorial-native-authentication-android-sign-up
+                    displayDialog(getString(R.string.unexpected_sdk_result_title), actionResult.toString())
+                }
                 is SignUpResult.PasswordRequired -> {
+                    // Please double-check the user flow settings of the application (client id) to make sure it's email + otp.
```

Comment:
Similar comment applies here too:
```suggestion
                     // Please refer to the EmailPasswordSignInSignUpFragment for handling sign in with password.
```